development/sentry-upload-artifacts.sh

```diff
@@ -51,6 +54,16 @@ function main {
         release="${2}"
         shift
         ;;
+      --dist-directory)
+        if [[ -z $2 ]]
+        then
+          printf "'dist-directory' option requires an argument.\\n" >&2
+          printf '%s\n' "${__SEE_HELP_MESSAGE__}" >&2
+          exit 1
+        fi
+        dist_directory="${2}"
```

Comment:
Should we bother assigning `dist_directory` to `"dist"` if we require it anyway?

Reply:
Sorry I don't understand what you mean

Reply:
Oh, if you mean to suggest that it's a required parameter... it is not, it's optional. `dist` is the detault.